### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by k4r7u** [lukaszgajos/keepasspasswordmanager#1]
Thanks for the awesome app.
It would be great though, if you could push the app to F-Droid as well.

**Comment by lukaszgajos** [comment]
I'll add app to F-Droid but after next update. Few nice features is coming, dropbox and many tiny fixes.

**Comment by k4r7u** [comment]
Thanks. Will WebDav/owncloud support be added in the future?

**Comment by lukaszgajos** [comment]
Yep, but all of external integration should be done in other way. Unfortunately android storage framework it's not as good as I was thought. Each cloud storage provider works in different way.
